gobrew_test.go

```diff
@@ -30,20 +30,22 @@ func TestJudgeVersion(t *testing.T) {
 		},
 		{
 			version:     "1.18@latest",
-			wantVersion: "1.18.6",
+			wantVersion: "1.18.7",
 		},
 		{
 			version:     "1.18@dev-latest",
-			wantVersion: "1.18.6",
-		},
-		{
-			version:     "latest",
-			wantVersion: "1.19.1",
-		},
-		{
-			version:     "dev-latest",
-			wantVersion: "1.19.1",
+			wantVersion: "1.18.7",
 		},
```

Comment:
Why you removed latest and dev-latest tags?

Reply:
Since, the new versions on both those will always be changing, it is hard to test the exact version output from those. Therefore commented those tests out.

Reply:
👌🏼